app/models/flood_risk_engine/enrollment.rb

```diff
@@ -78,6 +78,12 @@ def state_machine
       withdrawn: 6        # BO: used to hide anything created in error
     }
 
+    def submit
+      return if submitted_at.present?
+      self.submitted_at = Time.zone.now
+      pending!
```

Comment:
I don't believe so, this should be a one off, and if pending needs setting in other circumstances can be fired individually - AFAIK this check wanted, to stop some issues found in WEX with states getting reset, when NCCC started editing enrollments